src/IronWorkerRuntime.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace IronWorker
+
+use IronCore\IronCore;
+
+/**
+ * PHP runtime library for IronWorker
+ * IronWorker is a massively scalable background processing system.
+ * Use this class to access common functionality from inside an IronWorker task
+ *
+ * @link https://github.com/iron-io/iron_worker_php
+ * @link http://www.iron.io/
+ * @link http://dev.iron.io/
+ * @version 2.0.2
```

Comment:
thinking about removing `@version` line from this file.
reason: it's 3rd place where `@version` is written, and there are ridiculously high chance to forget to update it on next version bump - which means this line will likely be outdated really soon